### PR TITLE
Add How to raise issues and share feedback

### PR DESCRIPTION
**Pull request by yndajas**
This was requested by Wendy, Director of Marketing and Communications

### CODE REVIEW

File: src/about-us/how-to-raise-issues-and-share-feedback.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### Operational issues
+
+Operational issues are things relating to the day-to-day running of the company, where you can expect to raise things as needed and receive timely feedback. We're doing it right if this feels like an open and respectful ongoing conversation about how the business is operating.
+
+The company will provide regular feedback on the main themes being raised by staff, any action being taken as a result and if not, why that's the case. The main channels for this feedback to staff will be dxw quarterly performance updates and the People Experience roadmap meetings. Additional updates will be provided via the Company weeknote and Bikesheds (internal blog posts), as and when needed.
+
+If you feel that you're not getting feedback on an issue you've raised, please let the Head of People know. They are responsible for making sure issues raised via different routes are recorded, have an allocated owner and are responded to in a timely and appropriate way.
+
+#### Who to approach about what
```
Comment (Gweaton) [inline src/about-us/how-to-raise-issues-and-share-feedback.md:70]: question/suggestion: is this supposed to be only relevant to Operational issues, or could it be an h3?

File: src/about-us/how-to-raise-issues-and-share-feedback.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+##### Directors
+
+The Directors operate at a more strategic level and have overall responsibility for the leadership of the company and its performance, with a particular focus on its commercial success, external profile and reputation, and staff engagement. They manage any major issues and risks. The Operational Leadership Team will escalate issues to the Directors when they need their input or support to make decisions or act on something.
+
+You can ask questions of the Directors yourself, or via others, at dxw monthly connect meetings, dxw quarterly performance update meetings and Finance clinics. All of the Directors have a (virtual) open door policy and are happy to be contacted via Slack or email if you think there is something they should be made aware of directly or you'd like to discuss with them.
+
+This is where there is the most likely potential overlap with the ‘owner feedback' raised via the Trustees. There are no hard and fast right answers here. You can raise things with both the Directors and Trustees if that makes sense. If you are uncomfortable going to Directors about a particular issue, approaching Trustees might be a good alternative.
+
+#### Policies
```
Comment (Gweaton) [inline src/about-us/how-to-raise-issues-and-share-feedback.md:102]: suggestion: as above, this could potentially be an h3 as well

**Review by Gweaton — APPROVED** [review]
Just two comments re: headings

File: src/about-us/how-to-raise-issues-and-share-feedback.md (hunk 1 above)
Comment (yndajas) [reply src/about-us/how-to-raise-issues-and-share-feedback.md:70]: Per Slack, I've confirmed the current nesting is correct with Wendy

File: src/about-us/how-to-raise-issues-and-share-feedback.md (hunk 2 above)
Comment (yndajas) [reply src/about-us/how-to-raise-issues-and-share-feedback.md:102]: Per Slack, I've confirmed the current nesting is correct with Wendy

**Pull request merged**